lib/memsize_helpers.rb

```diff
@@ -1,6 +1,40 @@
+require "objspace"
+
 module MemsizeHelpers
   def self.memsize_rss_in_kb
-    _, size = %x(ps -p "#{$$}" -o pid=,rss=).strip.split.map(&:to_i)
-    size
+    %x(ps -p "#{$$}" -o rss=).strip.to_i
+  end
+
+  def self.reset!
+    @memsize_rss_start = memsize_rss_in_kb
+  end
+  reset!
```

Comment:
The "leak*.rb" scripts had duplication and I appreciate you cleaning that up, but they also noted the initial heap size at a particular location. This change moves when we start measuring the heap and kinda hides where it starts. I'd prefer if we called `reset!` wherever we want to start tracking or introduce a new method that takes a block and wrap the section of code we want to measure with that.

Reply:
Sure thing.  When I pulled this code out of my benchmark I had already moved where the record happened.  I actually noticed that the leak scripts were different, but wasn't sure if the line of code between was important.
Happy to put `reset!` calls where it was initialized before.

Reply:
I put `reset!` calls where it was previously initialized in #3